quickstarts/explore-pipeline-quickstart.yaml

```diff
@@ -51,18 +50,18 @@ spec:
           Pipelines Operator's status to change from **Installing** to **Succeeded**.
       review:
         instructions: |-
-          #### To verify that the OpenShift Pipelines Operator is installed:
+          #### To verify that the Red Hat OpenShift Pipelines Operator is installed:
           1. From the **Operators** section of the navigation, go to the **Installed Operators** page.
-          2. Verify that the **OpenShift Pipelines Operator** appears in the list of Operators.
+          2. Verify that the **Red Hat OpenShift Pipelines** appears in the list of Operators.
```

Comment:
Shouldn't this be?
```suggestion
          2. Verify that the **Red Hat OpenShift Pipelines Operator** appears in the list of Operators.
```

Reply:
This is a manual verification step that users would do to verify if their installation is success or not, so the highlighted text should exactly match the operator display name in the UI. This applies to be below comment as well.


![image](https://user-images.githubusercontent.com/9964343/164623124-2ceb0688-83aa-4b99-84e8-ace16d0cb6c7.png)